webui/react/src/pages/TaskList.tsx

```diff
@@ -155,11 +155,11 @@ const TaskList: React.FC = () => {
         plural: '',
         sources: [] as number[],
       };
-      if (record.misc?.experimentIds) {
+      if (record.misc?.experimentIds && record.misc?.experimentIds.length !== 0) {
```

Comment:
suggestion: normalize in the decoder to make sure it's always a list and just check the length here.